backend/data_distro/api_tests/schema/validate_log.txt

```diff
@@ -0,0 +1,1456 @@
+{'verbosity': 1, 'settings': None, 'pythonpath': None, 'traceback': True, 'no_color': False, 'force_color': False, 'skip_checks': False, 'view': 'vw_findings', 'year': 2022, 'start': 0, 'end': 100, 'step': None}
+
```

Comment:
👇 

Because we validate API responses one-by-one, the validation error messages are organized by `'id'`. It will print the object id, then the object itself, then an errors section with an ordered list of validation error messages.